scripts/mysqld_safe.sh

```diff
@@ -978,9 +978,7 @@ then
     if [ -f /sys/kernel/mm/transparent_hugepage/enabled ]; then
       CONTENT_THP=$(cat /sys/kernel/mm/transparent_hugepage/enabled)
       STATUS_THP=0
-      set +e
       STATUS_THP=$(echo $CONTENT_THP | grep -cv "\[${thp_setting}\]")
```

Comment:
It is better to leave it in case errexit is enabled on the system

Reply:
Hi @EvgeniyPatlan, yes that makes sense. However, that raises another issue: if errexit is enabled by default on the system and the --thp-setting option is not set, the script will again fail to restart mysql. How about disabling errexit straight away at the beginning of the script?